moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -53,7 +53,8 @@ num_outgoing_halt_msgs_to_publish: 4
 
 ## Configure handling of singularities and joint limits
 lower_singularity_threshold:  17.0  # Start decelerating when the condition number hits this (close to singularity)
-hard_stop_singularity_threshold: 30.0 # Stop when the condition number hits this
+approaching_stop_singularity_threshold: 23.5 # Stop when the condition number hits this and we are continuing to approach singularity
```

Comment:
@henrygerardmoore can't this be deleted? I think Nathan added it earlier on in this PR. It's not in the `main` branch, not in his graphic, and doesn't seem to be used for anything in the code